src/main/java/Deadline.java

```diff
@@ -0,0 +1,12 @@
+public class Deadline extends Task {
+    protected String by;
+
+    public Deadline (String description, String by){
+        super(description);
+        this.by = by;
+    }
```

Comment:
Advisable to have a uniform spacing strategy, to enhance code readability.

```suggestion
    public Deadline (String description, String by) {
        super(description);
        this.by = by;
    }
```